classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, in order to set up associations correctly, you need to implement the relationship in both ways. Here's a [refresher](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) from a previous lesson. Currently, you can't assign a classroom to a student, you can only add students to the classroom. 🤓 🔧